server/service/src/sync/translations/invoice.rs

```diff
@@ -632,15 +629,17 @@ fn invoice_status(invoice_type: &InvoiceType, data: &LegacyTransactRow) -> Optio
             LegacyTransactStatus::Fn => InvoiceStatus::Verified,
             _ => return None,
         },
-        InvoiceType::InventoryAddition | InvoiceType::InventoryReduction | InvoiceType::Repack => {
-            match data.status {
-                LegacyTransactStatus::Nw => InvoiceStatus::New,
-                LegacyTransactStatus::Sg => InvoiceStatus::New,
-                LegacyTransactStatus::Cn => InvoiceStatus::Verified,
-                LegacyTransactStatus::Fn => InvoiceStatus::Verified,
-                _ => return None,
-            }
-        }
+        InvoiceType::InventoryAddition | InvoiceType::InventoryReduction => match data.status {
+            LegacyTransactStatus::Nw => InvoiceStatus::New,
+            LegacyTransactStatus::Sg => InvoiceStatus::New,
+            LegacyTransactStatus::Cn => InvoiceStatus::Verified,
+            LegacyTransactStatus::Fn => InvoiceStatus::Verified,
+            _ => return None,
+        },
+        InvoiceType::Repack => match data.status {
+            LegacyTransactStatus::Fn => InvoiceStatus::Verified,
+            _ => return None,
```

Comment:
So there shouldn't be a repack leaving OG that has a status different from 'fn'?

Could you please put the answer in a comment in the code? think this would be useful.

Was there a problem or is this just some refactor?

Reply:
Just a refactor. Repacks only have `sg` and `fn` statuses. mSupply has an issue opened to deal with `sg` repacks before letting the user migrate the store to an Open mSupply site. Probably through an alert asking them to finalise suggested repacks